packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -106,6 +112,57 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
         .subscribe(
           (response) => {
             if (response.status === 201) {
+              if (config.showServiceQueueFields) {
+                const status = statuses.find((data) => data.display.toLowerCase() === 'waiting').uuid;
+                const defaultPriority = priorities.find((data) => data.display.toLowerCase() === 'not urgent').uuid;
+                const queuePayload: QueueEntryPayload = {
+                  visit: {
+                    uuid: response.data.uuid,
+                  },
+                  queueEntry: {
+                    status: {
+                      uuid: status,
+                    },
+                    priority: {
+                      uuid: priority ? priority : defaultPriority,
+                    },
+                    queue: {
+                      uuid: service,
+                    },
+                    patient: {
+                      uuid: patientUuid,
+                    },
+                    startedAt: toDateObjectStrict(toOmrsIsoString(new Date())),
+                  },
+                };
+
+                saveQueueEntry(queuePayload, abortController)
+                  .pipe(first())
+                  .subscribe(
+                    (response) => {
+                      if (response.status === 201) {
+                        showToast({
+                          kind: 'success',
+                          title: t('startVisit', 'Start a visit'),
+                          description: t(
+                            'startVisitQueueSuccessfully',
+                            'Patient has been added to active visits list and queue.',
+                            `${hours} : ${minutes}`,
```

Comment:
Would this read better if it said something like: `Patient added to the Clinical Consultation queue successfully`?